EssentialCSharp.Web/appsettings.json

```diff
@@ -1,12 +1,15 @@
 {
-    "Logging": {
-        "LogLevel": {
-            "Default": "Warning",
-            "Microsoft.AspNetCore": "Warning"
-        }
-    },
-    "AllowedHosts": "*",
-    "HCaptcha": {
-        "Secret": "0x0000000000000000000000000000000000000000" // Overwrite them with yours
+  "Logging": {
+    "LogLevel": {
+      "Default": "Warning",
+      "Microsoft.AspNetCore": "Warning"
     }
-}
+  },
+  "AllowedHosts": "*",
+  "HCaptcha": {
+    "Secret": "0x0000000000000000000000000000000000000000"
+  },
+  "ConnectionStrings": {
+    "EssentialCSharpWebContextConnection": "Server=(localdb)\\mssqllocaldb;Database=EssentialCSharp.Web;Trusted_Connection=True;MultipleActiveResultSets=true"
```

Comment:
Should this actually just be in the Development settings?

Reply:
I'm not sure what the issue is with it being here?

Reply:
Moved